tap_facebook/__init__.py

```diff
@@ -268,6 +268,7 @@ def sync_batches(self, stream_objects):
         api_batch.execute()
 
     key_properties = ['id']
+    replication_method = 'FULL_TABLE'
```

Comment:
Is it possible to define it in the `Stream` class?

Reply:
Yes, Defined the FULL_TABLE replication key in the `Stream` class removed from the extended class.